### PR TITLE
Adds Sea of Thieves Detection

### PR DESCRIPTION
**Pull request by sirvibegodlol**
the title

### CODE REVIEW

**Comment by GMatrixGames** [comment]
"Sea Of Thieves" is not a valid EGL game.

File: FModel/ViewModels/GameSelectorViewModel.cs [hunk 1]
```diff
@@ -64,6 +64,7 @@ private IEnumerable<DetectedGame> EnumerateDetectedGames()
             yield return GetRiotGame("VALORANT", "ShooterGame\\Content\\Paks");
             yield return new DetectedGame {GameName = "Valorant [LIVE]", GameDirectory = Constants._VAL_LIVE_TRIGGER};
             yield return GetMojangGame("MinecraftDungeons", "\\dungeons\\dungeons\\Dungeons\\Content\\Paks");
+            yield return GetUnrealEngineGame("Sea Of Thieves", "\\Athena\\Content\\Paks")
```
Comment (GMatrixGames) [inline FModel/ViewModels/GameSelectorViewModel.cs:67]: Not a valid Epic Games Store game.

**Review by GMatrixGames — COMMENTED** [review]
It's the main point really.

**Comment by ryantheleach** [comment]
@GMatrixGames I don't see anything on the project description that says that it's exclusive to EGS titles... But it's not that hard to add manually anyway.

**Comment by GMatrixGames** [comment]
At the time that this PR was made, FModel only supported detection of EGL, MC Dungeons, and Riot Games games. Since then, Steam support has been added.

This PR would need to be remade with the proper implementation for it to be accepted.

**Comment by ryantheleach** [comment]
Ah I see the issue now, the method name of GetUnrealEngineGame was misleading, as it's actually fetching epic game launcher games, not unreal engine games.

**Pull request closed without merging**